### ISSUE TITLE
[Documentation BUG] resource intersight_vnic_iscsi_adapter_policy

### ISSUE THREAD

**Issue by scotttyso** [CiscoDevNet/terraform-provider-intersight#105]
**Describe the bug**
The example shown for resource intersight_vnic_iscsi_adapter_policy has erroneous attributes in it.  burst, cos, max_data_field_size, and rate_limit are not applicable to this resource type.

```
resource "intersight_vnic_iscsi_adapter_policy" "vnic_iscsi_adapter_policy" {
  name                = "vnic_iscsi_adapter_policy1"
  description         = "vnic iscsi adapter policy"
  rate_limit          = 5
  burst               = 4000
  max_data_field_size = 512
  cos                 = 5
  organization {
    object_type = "organization.Organization"
    moid        = var.organization
  }
}
```

Should be

```
resource "intersight_vnic_iscsi_adapter_policy" "vnic_iscsi_adapter_policy" {
  connection_time_out   = 15
  description           = "vnic iscsi adapter policy"
  dhcp_timeout          = 60
  lun_busy_retry_count  = 15
  name                  = "vnic_iscsi_adapter_policy1"
  organization {
    object_type = "organization.Organization"
    moid        = var.organization
  }
}
```

**Comment by scotttyso** [comment]
Adding some more documentation bugs.

https://registry.terraform.io/providers/CiscoDevNet/intersight/latest/docs/resources/fabric_switch_control_policy

recovery_action:(string) UDLD recovery when enabled, attempts to bring an UDLD error-disabled port out of reset.* none - The standard 4th generation UCS Fabric Interconnect with 54 ports.* reset - The expanded 4th generation UCS Fabric Interconnect with 108 ports.

Should probably be 
recovery_action:(string) UDLD recovery when enabled, attempts to bring an UDLD error-disabled port out of reset.* none - No action is taken when a port has been disabled.* reset - The switch will attempt to bring a UDLD error-disabled back online.

https://registry.terraform.io/providers/CiscoDevNet/intersight/latest/docs/resources/fabric_link_aggregation_policy

Flag used to indicate whether LACP PDUs are to be sent 'fast', i.e., every 1 second.* normal - The expanded 4th generation UCS Fabric Interconnect with 108 ports.* fast - The standard 4th generation UCS Fabric Interconnect with 54 ports.

Should probably be something like
Flag used to indicate whether LACP PDUs are to be sent 'fast', i.e., every 1 second.* normal - The normal timeout rate is 30 seconds.* fast - The fast timeout rate is 1 second.

**Comment by scotttyso** [comment]
https://registry.terraform.io/providers/CiscoDevNet/intersight/latest/docs/resources/fabric_uplink_role

Example shows ud_ld_admin_state = "Enabled".  That is not a valid option for uplinks.

**Comment by scotttyso** [comment]
https://registry.terraform.io/providers/CiscoDevNet/intersight/latest/docs/resources/fabric_fc_uplink_role

One very small mistake on the example with vsan_id showing as vsanid

```
resource "intersight_fabric_fc_uplink_role" "fabric_fc_uplink_role1" {
  aggregate_port_id = 0
  slot_id           = 100
  admin_speed       = "Auto"
  fill_pattern      = "Idle"
  vsanid            = 10
}
```

Should be

```
resource "intersight_fabric_fc_uplink_role" "fabric_fc_uplink_role1" {
  aggregate_port_id = 0
  slot_id           = 100
  admin_speed       = "Auto"
  fill_pattern      = "Idle"
  vsan_id            = 10
}
```

**Comment by scotttyso** [comment]
https://registry.terraform.io/providers/CiscoDevNet/intersight/latest/docs/resources/vnic_iscsi_boot_policy#iscsi_adapter_policy

```
resource "intersight_vnic_iscsi_boot_policy" "vnic_iscsi_boot_policy" {
  name                           = "vnic_iscsi_boot_policy1"
  description                    = "vnic iscsi boot policy"
  dhcp_timeout                   = 300
  connection_time_out            = 300
  lun_busy_retry_count           = 300
  initiator_ip_source            = "DHCP"
  target_source_type             = "Auto"
  initiator_static_ip_v4_address = "10.1.1.1"
  chap = {
    object_type = "vnic.IscsiAuthProfile"
    password    = "ChangeMe"
    user_id     = "user_1"
  }
  mutual_chap = {
    object_type = "vnic.IscsiAuthProfile"
    password    = "ChangeMe"
    user_id     = "user_1"
  }
  organization {
    object_type = "organization.Organization"
    moid        = var.organization
  }
}
```

These are not valid attributes

  dhcp_timeout                   = 300
  connection_time_out            = 300
  lun_busy_retry_count           = 300

They should be removed

**Comment by scotttyso** [comment]
https://registry.terraform.io/providers/CiscoDevNet/intersight/latest/docs/resources/vnic_fc_qos_policy

Priority can only be FC

This needs to be changed

priority:(string)(Computed) The priortity matching the System QoS specified in the fabric profile.* Best Effort - QoS Priority for Best-effort traffic.* FC - QoS Priority for FC traffic.* Platinum - QoS Priority for Platinum traffic.* Gold - QoS Priority for Gold traffic.* Silver - QoS Priority for Silver traffic.* Bronze - QoS Priority for Bronze traffic.

**Comment by code-lucidal58** [comment]
The changes are included. Examples directory is open to the community for changes. Feel free to raise PR to add/update the examples.

**Comment by code-lucidal58** [comment]
Issue fixed in 1.0.15 release
